var/spack/repos/builtin/packages/py-pyusb/package.py

```diff
@@ -0,0 +1,18 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class PyPyusb(PythonPackage):
+    """Easy USB access from Python."""
+
+    homepage = "https://github.com/pyusb/pyusb"
+    pypi     = "pyusb/pyusb-1.0.2.tar.gz"
+
+    version('1.0.2', sha256='4e9b72cc4a4205ca64fbf1f3fff39a335512166c151ad103e55c8223ac147362')
+
+    depends_on('py-setuptools', type=('build', 'run'))
+    depends_on('libusb', type=('build', 'run'))
```

Comment:
Wouldn't:
```suggestion
    depends_on('libusb')
```
fit better?

Reply:
It looks for `libusb` at runtime. I think the dependencies are correct as is

https://github.com/pyusb/pyusb/blob/master/usb/libloader.py

Reply:
I think this is still `link` per our semantics. `run` is meant for runtime dependencies, like system calls to other tools, that have a separate process space and thus don't need strict unification of symbols. An `ldloaded` library is still `link`. Checking with @becker33 and @tgamblin if I'm saying the correct thing here.

Reply:
👍 I assumed that `link` does not affect the run environment and everything needs to be "compiled into the binary" (for values of compiled and binary). We wouldn't see that distinction as we are (somewhat manually) provide an environment with all (build, link and run) dependencies

Reply:
Done